compiler/ast.nim

```diff
@@ -1026,6 +1026,26 @@ const
   defaultAlignment = -1
   defaultOffset = -1
 
+
+proc getnimblePkg*(a: PSym): PSym =
+  result = a
+  while result != nil:
+    case result.kind
+    of skModule:
+      result = result.owner
+      assert result.kind == skPackage
+    of skPackage:
+      if result.owner == nil:
+        break
+      else:
+        result = result.owner
```

Comment:
Do we have "packages of packages"? I hope we don't. make this an ``assert result.owner == nil``.